generate-docs/tools/coverage-tester.ts

```diff
@@ -268,9 +268,9 @@ function averageDescriptionRatings(ratings: DescriptionRating[]) : DescriptionRa
 function convertToCsv(apiCoverage: Map<string, ClassCoverageRating>) : string {
     let csvString = "Class,Field,Type,Description Rating, Has Example?\n";
     apiCoverage.forEach((coverage, className) => {
-        csvString += `${className},,${coverage.classRating.type},${coverage.classRating.descriptionRating},${coverage.classRating.hasExample}\n`;
+        csvString += `${className},<class description>,${coverage.classRating.type},${coverage.classRating.descriptionRating},${coverage.classRating.hasExample}\n`;
```

Comment:
Per our discussion, can we update to "not applicable" or something similar?
```suggestion
        csvString += `${className},not applicable,${coverage.classRating.type},${coverage.classRating.descriptionRating},${coverage.classRating.hasExample}\n`;
```

Reply:
I did "N/A" instead :-)